src/main/java/com/fasterxml/jackson/annotation/JsonFormat.java

```diff
@@ -261,6 +261,9 @@ public enum Feature {
          */
         ACCEPT_CASE_INSENSITIVE_PROPERTIES,
 
+        READ_UNKNOWN_ENUM_VALUES_AS_NULL,
```

Comment:
needs javadoc - also needs `@since 2.15` in that javadoc

Reply:
Thanks for looking into the PR. I have added javadoc with `@since 2.15` .